### DISCUSSION TITLE
OpenAPI block - URL does not work for new blocks

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#789] [0. General bugs and questions]
I have a developer who is experimenting with the OpenAPI block functionality.

However, when he tries to specify a URL for the API, it fails with an unknown error.

The URL for the API is publicly accessible.

Quote from the developer - "we can't load or openapi docs from the URL but the exact same json file works fine when loaded from a file"

**Comment by rodrcastro** [comment]
Hey @JaniceManwiller 👋 Thank you for your patience on this :) Are you testing this out in a change request or in a public documentation? Can you please share with us at support@gitbook.com the URL for where you are testing this out so we can take a closer look on our side as well? We want to see what errors are shown and confirm this. Appreciate your help on this! :)

**Reply by akamor** [reply]
Hi, I work with Janice and am on the engineering team.  The URL we are trying to use is:

https://textual.tonic.ai/openapi/openapi.json

When I hit this URL from my browser it serves the JSON spec as expected.  Also, issuing a plain GET request from Postman also retrieves the resource as expected.
